cmd/bee/cmd/start.go

```diff
@@ -80,34 +82,31 @@ func (c *command) initStartCmd() (err error) {
 			}
 
 			beeASCII := `
-            Welcome to Swarm.... Bzzz Bzzzz Bzzzz
-                            \     /
-                        \    o ^ o    /
-                          \ (     ) /
-               ____________(%%%%%%%)____________
-              (     /   /  )%%%%%%%(  \   \     )
-              (___/___/__/           \__\___\___)
-                 (     /  /(%%%%%%%)\  \     )
-                  (__/___/ (%%%%%%%) \___\__)
-                          /(       )\
-                        /   (%%%%%)   \
-                             (%%%)
-                               !                   `
+Welcome to Swarm.... Bzzz Bzzzz Bzzzz
+                \     /
+            \    o ^ o    /
+              \ (     ) /
+   ____________(%%%%%%%)____________
+  (     /   /  )%%%%%%%(  \   \     )
+  (___/___/__/           \__\___\___)
+     (     /  /(%%%%%%%)\  \     )
+      (__/___/ (%%%%%%%) \___\__)
+              /(       )\
+            /   (%%%%%)   \
+                 (%%%)
+                   !                   `
 
 			fmt.Println(beeASCII)
 			fmt.Print(`
-
 DISCLAIMER:
 This software is provided to you "as is", use at your own risk and without warranties of any kind.
 It is your responsibility to read and understand how Swarm works and the implications of running this software.
 The usage of Bee involves various risks, including, but not limited to:
 damage to hardware or loss of funds associated with the Ethereum account connected to your node.
 No developers or entity involved will be liable for any claims and damages associated with your use,
-inability to use, or your interaction with other nodes or the software.
-
-`)
+inability to use, or your interaction with other nodes or the software.`)
 
-			logger.Infof("version: %v", bee.Version)
+			fmt.Printf("\n\nversion: %v - supported until %v\n\n", bee.Version, endSupportDate())
```

Comment:
Not surprise people when the version is not supported anymore due to a critical bug and give people a link to our official channel:

`planned to be supported until date, please follow http://ethswarm.org/`

Reply:
Feel free to word it different, or separate these sentences